colcon-build/action.yaml

```
@@ -51,7 +55,7 @@ runs:
       run: |
         package_paths=$(colcon list -p --packages-above-and-dependencies ${{ inputs.target-packages }} --base-paths . dependency_ws)
         sudo apt-get -yqq update
-        rosdep update
+        rosdep update ${{ inputs.rosdep-update-arguments }}
```

If you go https://github.com/autowarefoundation/autoware-github-actions/pull/207/files#r1119493824, for example, this part will be as follows:

```suggestion
        if [ "${{ inputs.include-eol-distros }}" = "true" ]; then
            rosdep update --include-eol-distros
        else
            rosdep update
        fi
```